core/web/presenters/eth_key.go

```diff
@@ -33,12 +33,12 @@ func (r ETHKeyResource) GetName() string {
 
 // NewETHKeyOption defines a functional option which allows customisation of the
 // EthKeyResource
-type NewETHKeyOption func(*ETHKeyResource) error
+type NewETHKeyOption func(*ETHKeyResource)
 
 // NewETHKeyResource constructs a new ETHKeyResource from a Key.
 //
 // Use the functional options to inject the ETH and LINK balances
-func NewETHKeyResource(k ethkey.KeyV2, state ethkey.State, opts ...NewETHKeyOption) (*ETHKeyResource, error) {
+func NewETHKeyResource(k ethkey.KeyV2, state ethkey.State, opts ...NewETHKeyOption) *ETHKeyResource {
```

Comment:
Constructing a `NewETHKeyResource` can no longer fail. It is the `NewETHKeyOption` implementation's responsibilty to handle them - in this case log and leave the fields nil.